R/make_mixture_from_samples.R

```diff
@@ -123,6 +126,6 @@ mixture_from_samples <- function(data,
     out[, geography := NULL]
   }
   
-  return(out[])
+  return(out)
```

Comment:
I think the [] needs to be there. This was the issue I messaged you about where you can't access whatever the function returns directly and this was a suggested solution

Reply:
I don't think you want to do that unless you are wanting to print it. `data.table` doesn't print on purpose until it is called twice and it is very odd to put this wrapper on it to alter that behaviour.

Reply:
okk. agreed :)